### ISSUE TITLE
Multimonitor settings

### ISSUE THREAD

**Issue by 7OH** [irzyxa/Volume2#120]
**Describe the bug**
![Снимок экрана (37)](https://user-images.githubusercontent.com/3063507/103698452-0b723200-4faa-11eb-9901-1b7afbcfbf2d.png)
![Снимок экрана (38)](https://user-images.githubusercontent.com/3063507/103698457-0ca35f00-4faa-11eb-8037-dbef6b9d57de.png)

**Desktop (please complete the following information):**
Volume2 version: 1.1.6.428
Windows version: Windows 10
WOW64: True
Version: Portable

**Additional context**
Popup is set for right-bottom corner without taskbar.
On main monitor it shows correct, on secondary it show popup not at corner.
If we must\can set settings for all monitors separately - then will be logical to have a list of monitors in setting and see settings for every.
After dragging settings window to second monitor and set config on second monitor - it works fine,
Not every user can guess this behavior.

**Comment by irzyxa** [comment]
Yes, here user should setup each monitor and than select active window. I do not know how it will be better to implement it, if you have please write your thoughts.

**Comment by iDolmatov** [comment]
Думаю, что в первую очередь нужно правильно считывать размеры каждого из мониторов (экранов) и на основе этого рассчитывать позицию для каждого монитора. Отслеживать разрешения и активность мониторов, чтобы при подключении мониторов/смене разрешения (в т.ч.) при запуске полноэкранных игр в другом разрешении.

Во вторую очередь можно добавить опцию задания координат относительно разрешения монитора. Настраиваться будет также, как и сейчас, но вызывается список разрешений и у каждого свои настройки. 
Здесь вот лучше подумать делать это отдельной галочкой со списком, либо полностью заменить шестерёнку на список из вариантов "единые настройки для всех мониторов", "индивидуальные настройки для мониторов". 
Ещё можно предлагать указать не разрешение экрана, а его порядковый номер в настройка системы. Давно не подключал три монитора, поэтому не помню что там показывает система в настройках.

**Comment by irzyxa** [comment]
@iDolmatov дело не в разрешении и не позиции. Для настройки каждого из экранов нужно выбрать его из списка и настроить его параметры, они сохранятся и все будет работать. Просто в интерфейсе это не совсем интуитивно сделано, а как лучше я ещё не придумал.

**Comment by 7OH** [comment]
У монитора есть ид ?
Винда как-то сохраняет позицию окон для каждого монитора (на работе за разными сижу постоянно).
Можно показывать настройку в списке только тех, которые видны, а сохранять под ид.
Сейчас светит только монитор1 как на скрине, причем на втором мониторе тоже цифра 1.

**Comment by 7OH** [comment]
Если ИД нету - сохраняем в виде "Монитор_Н__РазрХ_РазрУ" (Монитор_1__1920_1080)
Оно и в списке будет видно для каких разрешений какие настройки и самому подсветить в списке текущие конфигурации будет не сложно.
Как вариант

**Comment by irzyxa** [comment]
Вся информация уже сохраняется, и у меня это не воспроизводится. Сделайте настройку каждого экрана. Если не сложно вышлите свой Config.ini

**Comment by 7OH** [comment]
До начала настройки всё работало корректно.
Настроил на одном мониторе - сохранил (чтобы справа снизу и без панели) - показывает на том мониторе где настраивал поверх панели.
Сохранил настройки на двух - вроде работает.
Вышел из программы - перезапустил.
Теперь на одном показывает (и почему-то звук появился при регулировке).
Над вторым вообще не всплывает.
Конфиг вложил.
[Config.zip](https://github.com/irzyxa/Volume2/files/5991455/Config.zip)
